### PR TITLE
Remove unused dependencies from Tapyrus-core

### PR DESCRIPTION
**Pull request by Naviabheeman**
These commits from bitcoin core have been ported to tapyrus in this PR. 

|library|commit|
|--|--|
|dbus| e8fabd9|
|fontconfig | 86beb8c
libICE |7177e09
libSM|7177e09
libX11|0c55d8b
libXau|e439388
libxcb|5129b36, 173ef89, 3303124
libXext|9245699
Native_biplist|7087440
cdrkit|7587d11, 4acbcfa
Native_libdmg-hfsplus|9192ce5, cb9e88e, 04a69c2
qrencode|6a8ada3
qt|2654927, ddbb419, 430bf6c
xcb_proto|e97f5c1
xextproto|aa53cb7
xproto|0072237
xtrans|aa53cb7
zeromq|c295cba
zlib|e674e94

### CODE REVIEW

File: test/functional/test_framework/test_node.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def wait_for_rpc_connection(self):
                 self.url = self.rpc.url
                 self.log.debug("RPC successfully started")
                 self.time_to_connect = datetime.now() - start_time
-                return
+                return self.time_to_connect
```
Comment (azuchi) [inline test/functional/test_framework/test_node.py:173]: This change does not appear to be related to the intention of the PR.

File: src/qt/tapyrusgui.h [hunk 2]
```diff
@@ -49,15 +49,15 @@ QT_END_NAMESPACE
...
 {
     Q_OBJECT
 
 public:
     static const std::string DEFAULT_UIPLATFORM;
 
-    explicit BitcoinGUI(interfaces::Node& node, const PlatformStyle *platformStyle, const NetworkStyle *networkStyle, QWidget *parent = 0);
-    ~BitcoinGUI();
+    explicit TapyrusGUI(interfaces::Node& node, const PlatformStyle *platformStyle, const NetworkStyle *networkStyle, QWidget *parent = 0);
+    ~TapyrusGUI();
```
Comment (azuchi) [inline src/qt/tapyrusgui.h:60]: This rename change is not directly related to this PR.  PR should include only one basic intended change.

**Review by azuchi — CHANGES_REQUESTED** [review]

File: test/functional/test_framework/test_node.py (hunk 1 above)
Comment (Naviabheeman) [reply test/functional/test_framework/test_node.py]: Yes, there were some unrelated commits in this PR. I cleaned up the branch log. Now it should be correct

File: src/qt/tapyrusgui.h (hunk 2 above)
Comment (Naviabheeman) [reply src/qt/tapyrusgui.h:60]: without this change Tapyrus-Qt did not compile. I'll check if this can be split

Comment (Naviabheeman) [reply src/qt/tapyrusgui.h]: was able to remove this

**Review by azuchi — APPROVED** [review]
